tasks/get.yaml

```diff
@@ -1,220 +1,33 @@
 ---
-- environment:
-    AWS_ACCESS_KEY_ID: "{{ pool_manager_aws_access_key_id }}"
-    AWS_SECRET_ACCESS_KEY: "{{ pool_manager_aws_secret_access_key }}"
+# First, try to get the information from the sandbox API using uuid and Placement
+- when: sandbox_api_login_token is defined
```

Comment:
Note that the `defaults.yaml` is currently defining `sandbo_api_login_token` as empty string...